migrations/Version20201216112814.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Starfleet Project.
+ *
+ * (c) Starfleet <msantostefano@jolicode.com>
+ *
+ * For the full copyright and license information,
+ * please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20201216112814 extends AbstractMigration
+{
+    public function up(Schema $schema): void
+    {
+        $this->abortIf('postgresql' !== $this->connection->getDatabasePlatform()->getName(), 'Migration can only be executed safely on \'postgresql\'.');
+
+        $this->addSql('DROP INDEX uniq_911533c8d1b862b8');
+        $this->addSql('ALTER TABLE conference DROP hash');
+        $this->addSql('ALTER TABLE conference RENAME COLUMN location TO city');
+        $this->addSql('ALTER TABLE conference ADD country VARCHAR(255) DEFAULT \'Undefined\' NOT NULL');
```

Comment:
I don't like this "Undefined". Let's use `null` is done for that

Reply:
I could but the field is not nullable. Should I set it to Nullable then ? Or maybe this will work work anyway ?

Reply:
if the country could be missing, so yes the field must be nullable